ajax/libs/noti.js/package.json

```diff
@@ -16,6 +16,6 @@
   "license": "GPL-2.0",
   "repository": {
     "type": "git",
-    "url": "https://github.com/j-l-n/noti.js.git"
+    "url": "https://github.com/needim/noty.git"
```

Comment:
@PeterDaveHello how should I do that, another pull request? Or can I do a new commit to this pull request somehow?  Sorry, this is a little new to me.